src/index.js

```diff
@@ -23,19 +22,32 @@ const ping = ({ url, timeout }) => {
         isOnline();
       }
     };
-
     xhr.open("GET", url);
     xhr.timeout = timeout;
     xhr.send();
   });
 };
 
+const pingAllUrls = async ({ urls, timeout }) => {
+  for (const url of urls) {
+    try {
+      const result = await ping({ url, timeout });
+      if (result === true) {
+        return result;
+      }
+    } catch (err) {
+      return false;
+    }
+  }
+  return false;
+};
+
 const propTypes = {
   children: PropTypes.node,
   onChange: PropTypes.func,
   polling: PropTypes.oneOfType([
     PropTypes.shape({
-      url: PropTypes.string,
+      urls: PropTypes.array,
```

Comment:
it might be nice if this prop could be a string OR an array. that way the more common case is taken care of